site/en/blog/private-prefetch-proxy/index.md

```diff
@@ -0,0 +1,137 @@
+
+In the future, we hope to expand this feature to links with cookies or local state while maintaining its privacy characteristics. The challenge with cookies is that they might be used to alter the user experience in hard to predict ways. So, website owners will most likely have to opt in or adjust their site to benefit from private prefetch proxy for links with cookies.
+
+Concretely, while the prefetch requests will remain uncredentialed, the web page will gain access to cookies, and other local state, when the user navigates to it. Developers could take advantage of this to add back the personalization and changes based on cookies or local state. Or perhaps, developers might also be interested in declaring certain resources as perfectly fine to prefetch and use as is, without cookies (that is, resources that don't depend on any cookies). Please have a look at the [What's next](#next) section to learn more and inform our plan.
+
+### Geo-dependent content or services
+
+If your website behaves differently (for example, different content, or selective access) across markets based on the user's IP address, you may wonder how to handle private prefetch proxy's prefetch requests. It's important to know that private prefetch proxy is powered by several servers spread across the globe, and that the IP of the proxy will geolocate to the country from which the user initiated a prefetch.
+
+So, with that in mind, here is what we recommend:
+
+1. Identify prefetch requests from a private prefetch proxy by the presence of a Sec-Purpose: Prefetch; anonymous-client-ip HTTP header.
+1. Lookup the geolocation of the private prefetch proxy that issued the request via its IP address. See [this resource](https://www.gstatic.com/chrome/prefetchproxy/prefetch_proxy_geofeed) for an up-to-date list of rolled-out geographies and the corresponding IP addresses.
+1. Serve resources in accordance with the market attached to this particular geolocation.
+
+### Traffic control {: #traffic }
+
+From past experiments, we know that this feature typically results in less than 2% extra requests for main resources (for example HTML documents). That said, if you are the cautious kind, you can use the traffic advice's fraction field to control how much the private prefetch proxy should let through. You can start with a small fraction such as 0.3 (that is,. 30%), and gradually increase it to 1.0 (that is, 100%).
+
+```json
+[{
+    "user_agent": "prefetch-proxy",
+    "google_prefetch_proxy_eap": {
+        	"fraction": 0.3
+    }
+}]
+```
+
+The `fraction` field gives you control over the prefetch traffic by specifying how much the private prefetch proxy should let through. The value is a float between 0.0 (no prefetch at all) and 1.0 (100% of the prefetch requests get through).
+
+This _traffic advice file_ should be placed under the `/.well-known/` path of your website, and served with a MIME type of `application/trafficadvice+json`.
```

Comment:
I think this advice was given earlier.

Reply:
Yes, I also noted that on my side. We can remove the second mention.